discovery-provider/src/api/v1/helpers.py

```diff
@@ -179,6 +179,8 @@ def extend_track(track):
     owner_id = encode_int_id(track["owner_id"])
     if "user" in track:
         track["user"] = extend_user(track["user"])
+        if "slug" in track:
+            track["permalink"] = f'{track["user"]["handle_lc"]}/{track["slug"]}'
```

Comment:
interesting choice to do handle_lc. I would think people would want their casing if they had special casing?

Reply:
Oh whoops, that's actually a mistake!